plugins/woocommerce/client/admin/client/settings-payments/woopay-eligibility-store/resolvers.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import { getWooPayEligibility, setIsEligible } from './actions';
+import { WooPayEligibilityResponse } from './types';
+
+export function* getIsEligible(): Generator<
+	unknown,
```

Comment:
I couldn't get a granular type here 🤔

Reply:
I think it's not necessary to specify it, see https://github.com/woocommerce/woocommerce/blob/88610e6aaa0b6d01182afab9075ddb63af3135ad/packages/js/data/src/payment-settings/resolvers.ts#L17-L17